brayns/tasks/LoadDataFunctor.cpp

```diff
@@ -43,6 +47,25 @@ LoadDataFunctor::LoadDataFunctor(EnginePtr engine)
 
 void LoadDataFunctor::operator()(Blob&& blob)
 {
+    // extract the archive and treat it as 'load from folder'
+    if (isArchive(blob))
+    {
+        struct Scope
+        {
+            Scope(Blob&& blob, LoadDataFunctor& parent)
+            {
+                fs::create_directories(path);
+
+                extractBlob(std::move(blob), path.string());
+                parent._performLoad([&] { parent._loadData(path.string()); });
+            }
+
+            ~Scope() { fs::remove_all(path); }
+            fs::path path = "/tmp" / fs::unique_path();
```

Comment:
"/tmp" -> fs::temp_directory_path()? Even better, wouldn't it possible to process the archive in-memory, without writing and re-reading files?

Reply:
Thx for the first one, I was looking for that the whole time!
The latter is only possible if the archive only contains a single, because that's what the loaders support. Otherwise, a virtual filesystem is needed, where there is nothing trivial available.

Reply:
OK I see... I just remembered that libwebsockets has the ability to server contents directly from a zip file, but that's indeed using a virtual file system, which I don't know much about... probably too complicated for now..!